tests/test_for_support/test_for_consistency.py

```diff
@@ -725,6 +725,15 @@ def test_find_metabolites_consumed_with_closed_bounds(model, num):
     assert len(consumed) == num
 
 
+@pytest.mark.parametrize("model, num", [
+    ("textbook", 0),
```

Comment:
I cannot seem to find the definition of this `textbook` model anywhere, where is it defined?

In addition, it's smart to always test the two obvious cases (i.e. hypothesis and null-hypothesis). The function you're testing is supposed to find metabolites that cannot be produced when all exchanges are opened. So you ought to provide two cases, one where this is true i.e. there are metabolites that cannot be produced with open exchanges and one where this is false i.e. one where all metabolites can be produced.

Reply:
"textbook" is defined in `conftest.py`. Its the `EcoliCore.xml.gz` file, but I've changed this now with better tests as asked